data/xml/2022.eamt.xml

```diff
@@ -0,0 +1,881 @@
+      <author><first>Federico</first><last>Gaspari</last></author>
+      <author><first>Andre</first><last>K</last></author>
+      <author><first>Gauti</first><last>Kristmannsson</last></author>
+      <author><first>Jon Arild</first><last>Olsen</last></author>
+      <author><first>Natalia</first><last>Resende</last></author>
+      <author><first>Níels Rúnar</first><last>Gíslason</last></author>
+      <author><first>Dana D.</first><last>Sheridan</last></author>
+      <author><first>Páraic</first><last>Sheridan</last></author>
+      <author><first>John</first><last>Tinsley</last></author>
+      <author><first>Andy</first><last>Way</last></author>
+      <pages>347–348</pages>
+      <abstract>This paper provides an overview of the main achievements of the completed PRINCIPLE project, a 2-year action funded by the European Commission under the Connecting Europe Facility (CEF) programme. PRINCIPLE focused on collecting high-quality language resources for Croatian, Icelandic, Irish and Norwegian, which are severely low-resource languages, especially for building effective machine translation (MT) systems. We report the achievements of the project, primarily, in terms of the large amounts of data collected for all four low-resource languages and of promoting the uptake of neural MT (NMT) for these languages.</abstract>
+      <url hash="d8366f99">2022.eamt-1.64</url>
+      <bibkey>bago-etal-2022-achievements</bibkey>
+    </paper>
+    <paper id="65">
+      <title>Automatic Video Dubbing at <fixed-case>A</fixed-case>pp<fixed-case>T</fixed-case>ek</title>
+      <author><first>Mattia Di</first><last>Gangi</last></author>
+      <author><first>Nick</first><last>Rossenbach</last></author>
+      <author><first>Alejandro</first><last>Pérez</last></author>
+      <author><first>Parnia</first><last>Bahar</last></author>
+      <author><first>Eugen</first><last>Beck</last></author>
+      <author><first>Patrick</first><last>Wilken</last></author>
+      <author><first>Evgeny</first><last>Matusov</last></author>
+      <pages>349–350</pages>
+      <abstract>Video dubbing is the activity of revoicing a video while offering a viewing experience equivalent to the original video. The revoicing usually comes with a changed script, mostly in a different language, and the revoicing should reproduce the original emotions, coherent with the body language, and lip synchronized. In this project, we aim to build an AD system in three phases: (1) voice-over; (2) emotional voice-over; (3) full dubbing, while enhancing the system with human-in-the-loop capabilities for a higher quality.</abstract>
+      <url hash="ce1ff18c">2022.eamt-1.65</url>
+      <bibkey>gangi-etal-2022-automatic</bibkey>
+    </paper>
+    <paper id="66">
+      <title>Overview of the <fixed-case>ELE</fixed-case> Project</title>
+      <author><first>“Itziar</first><last>Aldabe</last></author>
+      <author><first>Jane</first><last>Dunne</last></author>
```

Comment:
weird stray quote here? was this in the source?

Reply:
FYI I think this may have been an issue from copying from/to Google Sheets. Apologies for that!